apps/docs/src/app/core/component-docs/card/examples/list-card/card-list-example.component.ts

```diff
@@ -1,7 +1,44 @@
-import { Component } from '@angular/core';
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, OnChanges, OnInit } from '@angular/core';
+import { FdSelectChange } from '@fundamental-ngx/core';
+
+interface ListData {
+    title?: string;
+    value?: number;
+    change?: number;
+}
 
 @Component({
     selector: 'fd-card-list-example',
-    templateUrl: 'card-list-example.component.html'
+    templateUrl: 'card-list-example.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class CardListExampleComponent {}
+export class CardListExampleComponent implements OnInit {
+    selectedValue = 'By Supplier';
+    options: string[] = ['By Supplier', 'By Zone'];
+    showData: ListData[] = [];
+
+    bySupplierData: ListData[] = [
+        { title: 'Delbont Industries', value: 91, change: 4 },
+        { title: 'Brazil Technologies', value: 52, change: 3 },
+        { title: 'Anav Ideon', value: 23, change: 1 },
+        { title: 'Loremt', value: 21, change: 1 }
+    ];
+
+    byZoneData: ListData[] = [
+        { title: 'North Zone', value: 91, change: 14 },
+        { title: 'West Zone', value: 80, change: 9 },
+        { title: 'South Zone', value: 45, change: 12 },
+        { title: 'east Zone', value: 21, change: 9 }
+    ];
+
+    constructor(private _cd: ChangeDetectorRef) {}
```

Comment:
why do we have ChangeDetectorRef in the documentation part?
Not all applications consuming our library are using onpush strategy. The code should work with and without it

Reply:
removed ChangeDetectorRef